NEWS.md

```diff
@@ -19,6 +20,70 @@ The format is based on [Keep a Changelog]
 | 2021-04-15 | [0.2.0]       | 0.13.0          | 5011          |
 | 2021-03-05 | [0.1.0]       | 0.12.0          | 4627          |
 
+## 0.7.0 - 2023-01-27
+
+As usual, this release contains lots of refactoring and bug fixes with
+particular thanks going to @Consolatis, @jlindgren90, @bi4k8, @Flrian and
+@Joshua-Ashton.
+
+### Added
+
+- Add `<regions>` config option allowing the definition of regions to which
+  windows can be snapped by keeping a keyboard modifier pressed while dragging
+  or by using the SnapToRegion action. Written-by: @Consolatis
+- Add `<SnapToRegion>` action. Written-by: @Consolatis
+- Add `<Kill>` action to send SIGTERM to a client process. Written-by: @bi4k8
+- Add config option `<core><reuseOutputMode>` to support flicker free boot
+  (issue #724). Written-by: @Consolatis
+- Enable single-pixel-buffer-v1
+- Improve debugging by adding argument to Debug action adding a label for the
+  layer popup tree. Written-by: @Consolatis
+- Support theme setting override by reading `<config-dir>/themerc-override`
+  where `<config-dir>` is normally $HOME/.config/labwc can be other locations
```

Comment:
Seems to miss an `but`.

Could also be reduced to just `Support single theme setting overrides by reading `<config-dir>/themerc-override`.

Reply:
Reduced.